apps/studio/components/interfaces/HomePageActions.tsx

```diff
@@ -62,14 +71,65 @@ const HomePageActions = ({
         </Button>
       )}
 
-      <Input
-        size="tiny"
-        placeholder="Search for a project"
-        icon={<IconSearch size={16} />}
-        className="w-64 [&>div>div>div>input]:!pl-7 [&>div>div>div>div]:!pl-2"
-        value={search}
-        onChange={(event) => setSearch(event.target.value)}
-      />
+      <div className="flex items-center gap-x-2">
+        <Input
+          size="tiny"
+          placeholder="Search for a project"
+          icon={<Search size={16} />}
+          className="w-64 [&>div>div>div>input]:!pl-7 [&>div>div>div>div]:!pl-2"
+          value={search}
+          onChange={(event) => setSearch(event.target.value)}
+        />
+
+        <Popover_Shadcn_>
+          <PopoverTrigger_Shadcn_ asChild>
+            <Button
+              type={filterStatus.length !== 2 ? 'secondary' : 'dashed'}
+              className="h-[26px] w-[26px]"
+              icon={<Filter />}
+            />
+          </PopoverTrigger_Shadcn_>
+          <PopoverContent_Shadcn_ className="p-0 w-56" side="bottom" align="center">
+            <div className="px-3 pt-3 pb-2 flex flex-col gap-y-2">
+              <p className="text-xs">Filter projects by status</p>
+              <div className="flex flex-col">
+                {[
+                  { key: 'ACTIVE_HEALTHY', label: 'Active' },
+                  { key: 'INACTIVE', label: 'Paused' },
+                ].map(({ key, label }) => (
+                  <div key={key} className="group flex items-center justify-between py-0.5">
+                    <div className="flex items-center gap-x-2">
+                      <Checkbox_Shadcn_
```

Comment:
Might need to wrap all this in a html form so that the checkbox labels are clickable

Reply:
@joshenlim

Reply:
oh are they not? i just checked on prod - the checkbox labels are clickable (which will toggle the checked state of the checkboxes)